examples/noisy_label_detection/trak_noisy_label.py

```diff
@@ -42,17 +49,18 @@ def get_mnist_indices_and_adjust_labels(dataset):
         sampler=SubsetSampler(range(1000)),
     )
 
-    model = train_mnist_lr(train_loader_full)
-    model.cuda()
+    model = train_cifar2_resnet9(train_loader, num_epochs=3, num_classes=10)
```

Comment:
maybe rename this function as train_cifar_resnet9?

Reply:
good catch. I have revised the module to be named as `dattri.benchmark.datasets.cifar` that contains both cifar2 and cifar10 functions.